lib/readify.js

```diff
@@ -31,24 +31,77 @@ const nicki = !WIN && !BROWSER && require('nicki/legacy');
 const readdir = promisify(fs.readdir, fs);
 
 /* sorting on Win and node v0.8.0 */
-const sortFiles = sort((a, b) => {
-    return a.name > b.name ? 1 : -1;
+const sortFiles = currify((attr, order, array) => {
+    const cmpCallbacks = {
+        'numeric': (a, b) => (+a - +b),
+        'local_string': (a, b) => a.localeCompare(b.attr),
+        'default': (a, b) => (a > b ? 1 : -1),
```

Comment:
Not much, it enforces the numeric cast with the `dir` value. But since I've switched formatting it maybe useless now.

localCompare is for accents. I'm french and I deal with accentuated letters like é è ë ê. Those letters have the same rank than e. Standard comparison put them after z.

http://www.jstips.co/en/sorting-strings-with-accented-characters/